src/Imdb/Title.php

```diff
@@ -1399,20 +1399,19 @@ public function top250()
     public function plot()
     {
         if (empty($this->plot_plot)) {
-            $page = $this->getPage("Plot");
-            if (empty($page)) {
+            $xpath = $this->getXpathPage("Plot");
+            //$page = $this->getPage("Plot");
+            if (empty($xpath)) {
                 return array();
             } // no such page
-            $doc = new \DOMDocument();
-            @$doc->loadHTML($page);
-            $xp = new \DOMXPath($doc);
-            $cells = $xp->query("//ul[@id=\"plot-summaries-content\"]/li[@id!=\"no-summary-content\"]");
+            $cells = $xpath->query("//ul[@id=\"plot-summaries-content\"]/li[@id!=\"no-summary-content\"]");
             foreach ($cells as $cell) {
                 $link = '';
-                if ($a = $cell->getElementsByTagName('a')->item(0)) {
+                $q = $cell->getElementsByTagName('a')->length - 1;
+                if ($a = $cell->getElementsByTagName('a')->item($q)) {
                     $href = preg_replace('!/search/title!i', 'https://' . $this->imdbsite . '/search/title',
                         $a->getAttribute('href'));
-                    $link = "\n-\n" . '<a href="' . $href . '">' . trim($cell->getElementsByTagName('a')->item(0)->nodeValue) . '</a>';
+                    $link = "\n-\n" . '<a href="' . $href . '">' . trim($cell->getElementsByTagName('a')->item($q)->nodeValue) . '</a>';
```

Comment:
As you assigning it to $a in the if above, you can just write it as $a->nodeValue.

Reply:
Jep you are right, it will be changed.